client/src/components/DatasetInformation/DatasetAttributes.vue

```diff
@@ -0,0 +1,138 @@
+                            <font-awesome-icon icon="cog" class="mr-1" />{{ "Convert" | l }}
+                        </template>
+                        <FormDisplay :inputs="result['conversion_inputs']" @onChange="onConversion" />
+                        <div class="mt-2">
+                            <b-button @click="submit('conversion', 'conversion')" variant="primary">
+                                <font-awesome-icon icon="exchange-alt" class="mr-1" />{{ "Create Dataset" | l }}
+                            </b-button>
+                        </div>
+                    </b-tab>
+                    <b-tab v-if="!result['datatype_disable']">
+                        <template v-slot:title>
+                            <font-awesome-icon icon="database" class="mr-1" />{{ "Datatypes" | l }}
+                        </template>
+                        <FormDisplay :inputs="result['datatype_inputs']" @onChange="onDatatype" />
+                        <div class="mt-2">
+                            <b-button @click="submit('datatype', 'datatype')" variant="primary" class="mr-1">
+                                <font-awesome-icon icon="save" class="mr-1" />{{ "Save" | l }}
+                            </b-button>
+                            <b-button @click="submit('datatype', 'datatype_detect')">
+                                <font-awesome-icon icon="redo" class="mr-1" />{{ "Auto-detect" | l }}
+                            </b-button>
+                        </div>
+                    </b-tab>
+                    <b-tab v-if="!result['permission_disable']">
+                        <template v-slot:title>
+                            <font-awesome-icon icon="user" class="mr-1" />{{ "Permissions" | l }}
+                        </template>
+                        <FormDisplay :inputs="result['permission_inputs']" @onChange="onPermission" />
+                        <div class="mt-2">
+                            <b-button @click="submit('permission', 'permission')" variant="primary">
+                                <font-awesome-icon icon="save" class="mr-1" />{{ "Save" | l }}
+                            </b-button>
+                        </div>
+                    </b-tab>
+                </b-tabs>
+            </div>
+        </DatasetAttributesProvider>
+    </div>
+</template>
+
+<script>
+import { getGalaxyInstance } from "app";
+import FormDisplay from "components/Form/FormDisplay";
+import { DatasetAttributesProvider } from "components/providers/DatasetProvider";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faBars, faCog, faDatabase, faExchangeAlt, faRedo, faSave, faUser } from "@fortawesome/free-solid-svg-icons";
+import { setAttributes } from "./services";
+
+library.add(faBars);
+library.add(faCog);
+library.add(faDatabase);
+library.add(faExchangeAlt);
+library.add(faRedo);
+library.add(faSave);
+library.add(faUser);
```

Comment:
Thanks a lot for your work @guerler!
That's just a very minor thing to save a tiny bit of space :sweat_smile: 
```suggestion
[faBars, faCog, faDatabase, faExchangeAlt, faRedo, faSave, faUser].forEach(icon => library.add(icon));
```

Reply:
@OlegZharkov No need for the function, you can just do:

```suggestion
library.add(faBars, faCog, faDatabase, faExchangeAlt, faRedo, faSave, faUser);
```

Reply:
oh, that's a good thing to know! Thx @dannon